gobblin-utility/src/main/java/org/apache/gobblin/util/jdbc/MysqlDataSourceUtils.java

```diff
@@ -15,7 +15,7 @@
  * limitations under the License.
  */
 
-package org.apache.gobblin.metastore.util;
+package org.apache.gobblin.util.jdbc;
```

Comment:
I understand the rename here, but seems after this change this will be inconsistent with the location. Should we move this class to another module?

Reply:
there was a file move in addition to the package rename.  AFAICT the current post-move location and package match